packages/dd-trace/src/telemetry/index.js

```diff
@@ -122,9 +129,9 @@ function start (aConfig, thePluginManager) {
 
   dependencies.start(config, application, host)
   sendData(config, application, host, 'app-started', appStarted())
+  heartbeat(config, application, host)
   interval = setInterval(() => {
     metricsManager.send(config, application, host)
```

Comment:
The `metricsManager.send(...)` should probably be moved into `heartbeat(...)` too. Not really a reason for those to be separate, and they are likely to just get out of sync if they're left separate.

Reply:
sounds good! will do

Reply:
actually if I move it now, it would break the unit test in a weird way. But it will be moved when we make the v2 PR, would it be okay if we keep it there for now?

Reply:
Yep, that's fine. 👍🏻